package.json

```diff
@@ -50,6 +50,7 @@
     "node-uuid": "^1.4.3",
     "page": "^1.6.4",
     "postcss": "^5.0.6",
+    "query-string": "^3.0.1",
```

Comment:
Is there a substantial difference between `query-string` and native node `querystring`?